### PR TITLE
feat: add dial metric to dashboard

### LINKED ISSUE #3745: Add new outgoing dials metric to grafana dashboard
### Describe the feature

new metric: `network_total_dial_successes` can be added to the grafana dashboard

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by i-m-aditya**
Fixes #3745 
Screenshot 2023-07-16 at 10 27 16 AM

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
lgtm, pending @Rjected

File: etc/grafana/dashboards/overview.json [hunk 1]
```diff
@@ -4769,6 +4769,99 @@
...
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "builder",
+          "expr": "reth_network_total_dial_successes",
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:4857]:
```suggestion
          "expr": "reth_network_total_dial_successes{instance=~\"$instance\"}",
```

**Review by Rjected — CHANGES_REQUESTED** [review]
Just one nit, otherwise looks good

File: etc/grafana/dashboards/overview.json [hunk 2]
```diff
@@ -4769,6 +4769,99 @@
       ],
       "title": "File Descriptors",
       "type": "timeseries"
+    },
+    {
```
Comment (Rjected) [inline etc/grafana/dashboards/overview.json:4773]: This should also probably be placed with the rest of the peer related metrics (for example after peer disconnect reasons)

**Comment by Rjected** [comment]
ping @i-m-aditya :)

**Comment by i-m-aditya** [comment]
@Rjected Apologies. I was occupied with office work last week. Will resolve comments today.

**Review by mattsse — APPROVED** [review]
ty!

pending @Rjected

**Review by onbjerg — APPROVED** [review]

**Review by Rjected — APPROVED** [review]
LGTM, thx!

**Pull request merged**